source/postprocess/memory_statistics.cc

```diff
@@ -37,13 +37,39 @@ namespace aspect
     {
       // memory consumption:
       const double mb = 1024*1024; // convert from bytes into mb
+
       statistics.add_value ("System matrix memory consumption (MB) ", this->get_system_matrix().memory_consumption()/mb);
       statistics.add_value ("Triangulation memory consumption (MB) ", this->get_triangulation().memory_consumption()/mb);
       statistics.add_value ("p4est memory consumption (MB) ", this->get_triangulation().memory_consumption_p4est()/mb);
-      statistics.add_value ("DoFHandler memory consumption (MB) ", this->get_dof_handler().memory_consumption()/mb);
-      statistics.add_value ("current_constraints memory consumption (MB) ", this->get_current_constraints().memory_consumption()/mb);
+
+      double dof_handler_mem = this->get_dof_handler().memory_consumption();
+      double constraints_mem = this->get_current_constraints().memory_consumption();
+      if (this->is_stokes_matrix_free())
+        {
+          dof_handler_mem += this->get_stokes_matrix_free().get_dof_handler_v().memory_consumption()
+                             + this->get_stokes_matrix_free().get_dof_handler_p().memory_consumption();
+#if DEAL_II_VERSION_GTE(9,2,0)
```

Comment:
what is the reason for this ifdef?

Reply:
`memory_consumption()` had not yet been implemented for DG DoFHandlers on the version of dealii the tester is run on.

Reply:
huh, interesting. Can you add this as a comment here, then?